demos/styles.yaml

```diff
@@ -257,7 +257,7 @@ layers:
             filter: water
 
         style:
-            # name: water
+            name: water
```

Comment:
Not sure why this change ended up here but could you revert? I turned water animation off by default in the main demo.